src/config.js

```diff
@@ -166,4 +167,8 @@ module.exports.setConfig = function setConfig(configData) {
     if (Number(configData.sendTimeout)) { // we do not allow 0 as a timeout
         config.sendTimeout = Number(configData.sendTimeout);
     }
+
+    if (configData.labels) {
+        config.labels = [...configData.labels];
```

Comment:
is it an array or an object? (the default is initialized as an object, but here you set it as an array)